src/Model/EventParticipation.php

```diff
@@ -50,7 +50,7 @@ public function setType($type)
     /** {@inheritDoc} */
     public static function getAvailableStatuses()
     {
-        return parent::getAvailableStatuses() + [static::STATUS_NONE, static::STATUS_ORGANIZER];
+        return array_merge(parent::getAvailableStatuses(), [static::STATUS_NONE, static::STATUS_ORGANIZER]);
```

Comment:
It works only for associative array in my case. BTW `array_merge` is explicit and I understand at the first what you want to do